src/block/Wheat.php

```diff
@@ -23,26 +23,34 @@
 
 namespace pocketmine\block;
 
+use pocketmine\block\utils\FortuneTrait;
 use pocketmine\item\Item;
 use pocketmine\item\VanillaItems;
-use function mt_rand;
+use function array_merge;
 
 class Wheat extends Crops{
+	use FortuneTrait;
 
-	public function getDropsForCompatibleTool(Item $item) : array{
-		if($this->age >= self::MAX_AGE){
-			return [
-				VanillaItems::WHEAT(),
-				VanillaItems::WHEAT_SEEDS()->setCount(mt_rand(0, 3))
-			];
-		}else{
+	public function getPickedItem(bool $addUserData = false) : Item{
+		return VanillaItems::WHEAT_SEEDS();
+	}
+
+	/**
+	 * @return Item[]
+	 */
+	protected function getFortuneDropsForLevel(int $level) : array{
+		if ($this->age >= self::MAX_AGE) {
+			return array_merge(
+				[VanillaItems::WHEAT()],
+				$this->binomialDrops(
+					VanillaItems::WHEAT_SEEDS(),
+					$level
+				)
```

Comment:
Perhaps `binomialDrops` could accept `array<Item> $extraDrops` to avoid this array_merge boilerplate everywhere.